cours/intro-awcc.en.md

```diff
@@ -2,14 +2,14 @@
 
 ### Training materials written by alter way Cloud Consulting
 
-<https://alterway.fr>
+ex Osones - <https://cloud-consulting.alterway.fr>
 
-![AWCC logo](images/logo-awcc.jpg){height="100px"}
+![AWCC logo](images/logo-awcc.jpg){height="70px"}
 
 - Copyright © 2014 - 2019 alter way Cloud Consulting
 - License: [Creative Commons BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/deed.fr)
 - Sources: <https://github.com/Osones/formations/>
-- HTML/PDF: <https://formation.alterway.fr/cloud/>
+- HTML/PDF: <https://osones.com/formations/>
```

Comment:
Should it be "https://cloud-consulting.alterway.fr/formations/" ?

Reply:
Nope, I don't think so. This is a corporate web page, with some links to some - potentially outdated - materials, whereas https://osones.com/formations/ is the exhaustive, up to date, list, of all up to date materials built by the CI.

Reply:
Oh yes it points directly to the S3 bucket !